docs/rules/jsx-max-props-per-line.md

```diff
@@ -58,4 +58,4 @@ The following patterns are not considered warnings:
 
 ## When not to use
 
-If you are not using JSX then you can disable this rule.
+If you are not using JSX or you already use the [`max-len` rule](http://eslint.org/docs/rules/max-len) then you can disable this rule.
```

Comment:
I can see how the max-len rule might overlap sometimes with this rule, but all the places I want to use this one wouldn't hit max-len.